### DISCUSSION TITLE
Getting newline characters output?

### DISCUSSION THREAD

**Discussion by samlhuillier** [aguingand/tiptap-markdown#68] [Q&A]
What config could I use so that typing enter in the editor translates to \n newline characters?

Right now, with this config:
```
Markdown.configure({
    html: true,
    tightLists: true,
    tightListClass: "tight",
    bulletListMarker: "-",
    linkify: true,
    breaks: true, // Disable the default behavior of converting newlines to <br>
    transformPastedText: false,
    transformCopiedText: false,
  });
```
typing lots of enters and calling getMarkdown results in only two newline characters.

Also, opening a file with lots of newline characters and calling set content like this:
```
editor?.commands.setContent(newFileContent, false, {
      preserveWhitespace: true,
    });
```
Results in only one newline in the editor. (I've tried setting breaks to false which didn't work either)

**Comment by danmindru** [comment]
did you figure this out @samlhuillier?
I have the same issue. I tried inserting double hard breaks but that doesn't create a new paragraph block. So it doesn't solve the problem of pasting text with > 1 line break, plus it makes styling a bit strange as setting the node to a heading would set it to multiple lines now.

I know it's non-standard markdown, but the result right now is what you see in the editor is not what you'll next time you open it due to \n being stripped.

Could we somehow write an extension that does that @aguingand?

**Reply by samlhuillier** [reply]
Nope and would absolutely love it if you could figure it out @danmindru
